tests/ovs_test.go

```diff
@@ -33,4 +37,51 @@ var _ = Describe("ovs-cni", func() {
 			})
 		})
 	})
+
+	Describe("pod ovs-bridge connectivity tests", func() {
+		Context("when an OVS bridge is configured on a node", func() {
+			const bridgeName = "br-test"
+			BeforeEach(func() {
+				node.AddOvsBridgeOnNode(bridgeName)
+			})
+			AfterEach(func() {
+				node.RemoveOvsBridgeOnNode(bridgeName)
+			})
+
+			Context("and a network attachment definition is defined", func() {
+				const nadName = "ovs-net"
+				BeforeEach(func() {
+					clusterApi.CreateNetworkAttachmentDefinition(nadName, bridgeName, `{ "cniVersion": "0.3.1", "type": "ovs", "bridge": "`+bridgeName+`", "vlan": 100 }`)
+				})
+				AfterEach(func() {
+					clusterApi.RemoveNetworkAttachmentDefinition(nadName)
+				})
+
+				Context("and two pods are connected through it", func() {
+					const (
+						pod1Name = "pod-test-1"
+						pod2Name = "pod-test-2"
+						cidrPod1 = "10.0.0.1/24"
+						cidrPod2 = "10.0.0.2/24"
+					)
+					BeforeEach(func() {
+						clusterApi.CreatePrivilegedPodWithIp(pod1Name, nadName, bridgeName, cidrPod1)
+						clusterApi.CreatePrivilegedPodWithIp(pod2Name, nadName, bridgeName, cidrPod2)
+					})
+					AfterEach(func() {
+						clusterApi.DeletePodsInTestNamespace()
+					})
+
+					Specify("they should be able to communicate over the network", func() {
+						By("Checking pods connectivity by pinging from one to the other")
+						ipPod2, _, err := net.ParseCIDR(cidrPod2)
+						Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("should succeed parsing pod's cidr: %s", cidrPod2))
+
+						err = clusterApi.PingFromPod(pod1Name, "test", ipPod2.String())
+						Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("should be able to ping from pod %s to IP %s which is on pod %s", pod1Name, ipPod2, pod2Name))
```

Comment:
suggestion: more information on ping failure: 
`fmt.Sprintf("should be able to ping from pod '%s@%s' to pod '%s@%s'", pod1.Name, pod1.IP, pod2.Name, pod2.IP`)

Reply:
[Done](https://github.com/kubevirt/ovs-cni/compare/2f89681ada6776e54cce3f2e1229bbbe20bd49c1..4c383b8b461f651e74455bf35659de11765ed1df)